packaging/linux/shotcut.1

```diff
@@ -0,0 +1,34 @@
+.TH SHOTCUT "1" "December 2018" "Shotcut 18.12.06" "User Commands"
+.SH NAME
+shotcut \- video editor
+.SH SYNOPSIS
+.B shotcut
+[\fI\,options\/\fR] \fI\,resource\/\fR
```

Comment:
"resource" can be multiple files and folders and is optional. [GNU ls uses `[FILE]...`](https://linux.die.net/man/1/ls). I can improve the usage output from the executable for consistency.
```
--- src/main.cpp
+++ src/main.cpp
@@ -144,8 +144,8 @@ public:
             QCoreApplication::translate("main", "The directory for app configuration and data."),
             QCoreApplication::translate("main", "directory"));
         parser.addOption(appDataOption);
-        parser.addPositionalArgument("resource",
-            QCoreApplication::translate("main", "A file to open."));
+        parser.addPositionalArgument("[FILE]...",
+            QCoreApplication::translate("main", "Zero or more files or folders to open"));
         parser.process(arguments());
 #ifdef Q_OS_WIN
         isFullScreen = false;
```